src/formatDate.js

```diff
@@ -8,7 +8,37 @@
  * @returns {string}
  */
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const separateDate = date.split(fromFormat[3]);
+  const newDate = [];
+  const separator = toFormat[3];
+
+  for (let i = 0; i < fromFormat.length - 1; i++) {
+    let index = toFormat.indexOf(fromFormat[i]);
+
+    if (index === -1) {
+      if (fromFormat[i].length === 2) {
+        index = toFormat.indexOf('YYYY');
+
+        if (separateDate[i] >= 30) {
+          separateDate[i] = '19' + separateDate[i];
+        }
+
+        if (separateDate[i] < 30) {
+          separateDate[i] = '20' + separateDate[i];
+        }
```

Comment:
The logic for determining the century based on the year is correct, but it could be simplified by using an else statement after checking if the year is greater than or equal to 30. This would make the code more efficient by avoiding a second comparison.